qiskit/circuit/library/__init__.py

```diff
@@ -354,8 +449,10 @@
    clifford_8_2
    clifford_8_3
 
-RZXGate template circuits
-=========================
+RZXGate template circuits (:mod:`qiskit.circuit.library.templates.rzx`)
+-----------------------------------------------------------------------
+
+Template circuits with :class:`~qiskit.circuit.library.standard_gates.rzx.RZXGate`.
```

Comment:
This cross-ref won't load; the path is based on the `currentmodule` in Sphinx at the time the object is documented, not the full Python path.  In this case, it's documented in this file, so the full path would be `qiskit.circuit.library.RZXGate`, but you can also just do "dot" lookup: `` :class:`.RZXGate` `` which invokes a search.

Reply:
done in 7a5292e8f1ebb6f3013a030cd6afe7f91261a787